src/components/Cards/Card.tsx

```diff
@@ -0,0 +1,19 @@
+import type { CardProps } from '../../type/card';
+
+import './Card.scss';
```

Comment:
프로젝트가 module scss를 사용하시는 것 같은데 일관성있게 바꾸면 좋을 것 같습니다!

```suggestion
import styles from './Card.scss';

<div className={styles.cardContent}>
```